src/components/LeftSidebar/SearchBox/SearchBox.vue

```diff
@@ -136,16 +136,23 @@ export default {
 		},
 
 		handleBlur(event) {
+			// Blur triggered by clicking on the trailing button
 			if (event.relatedTarget?.classList.contains('input-field__clear-button')) {
 				event.preventDefault()
 				this.getTrailingButton()?.addEventListener('blur', (trailingEvent) => {
 					this.handleBlur(trailingEvent)
 				})
-			} else {
-				this.$emit('blur', event)
-				if (this.value === '') {
-					this.$emit('update:is-focused', false)
-				}
+				return
+			}
+			// Blur triggered by clicking on a conversation item
+			const list = document.querySelector('.left-sidebar__list')
+			if (list && list.contains(event.relatedTarget)) {
```

Comment:
list can be a ref, passed as a prop (to not use native JS selectors - can break as depends on class):

```vue
<SearchBox :list="list" />
...
<li ref="list" class="left-sidebar__list">
...
setup() {
  const list = ref(null)
  return { list }
}
```

```suggestion
			if (list?.contains(event.relatedTarget)) {
```